<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shopList</class>
 <widget class="QWidget" name="shopList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="buylistWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>711</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="money_lb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>总价：¥</string>
   </property>
  </widget>
  <widget class="QLabel" name="money_tot">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>534</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(91, 141, 28);</string>
   </property>
   <property name="text">
    <string>结算</string>
   </property>
  </widget>
  <widget class="QLabel" name="fail_lb">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>550</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3 Prong Tree</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结算失败 ！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
